Modules/_lzmamodule.c

```diff
@@ -1314,15 +1316,33 @@ PyDoc_STRVAR(Decompressor_needs_input_doc,
 PyDoc_STRVAR(Decompressor_unused_data_doc,
 "Data found after the end of the compressed stream.");
 
+static PyObject *
+Decompressor_unused_data_get(PyObject *op, void *Py_UNUSED(ignored))
```

Comment:
```suggestion
Decompressor_unused_data_get(PyObject *op, void *Py_UNUSED(closure))
```

I always think that Py_UNUSED(ignored) is not helpful. It's more meaningful to indicate what the parameter was meant to represent.